main_f4.c

```diff
@@ -175,13 +182,78 @@ board_test_force_pin()
 	return false;
 }
 
+#if INTERFACE_USART
+static bool
+board_test_usart_receiving_break()
+{
+	// Start the timer - enable TIM2 clock
+	rcc_peripheral_enable_clock(&RCC_APB1ENR, RCC_APB1ENR_TIM2EN);
+
+	//Time base configuration
+	timer_set_prescaler(TIM2, 0); // Timer ticks at base clock speed (2 x APBPB1 = 84MHz)
```

Comment:
Currently I have hardcoded in the use of TIM2. Do we want to put this to be configurable in hw_config.h?